.github/workflows/BuildAndTest.yaml

```diff
@@ -1,50 +0,0 @@
-name: Build and Test
-
-on: [push, pull_request]
```

Comment:
This was causing the jobs to run twice per each PR update, so I changed it to just push